forui/lib/src/widgets/header/header.dart

```diff
@@ -1,30 +1,89 @@
+import 'package:flutter/foundation.dart';
 import 'package:flutter/widgets.dart';
 
 import 'package:forui/forui.dart';
+import 'package:forui/src/foundation/tappable.dart';
 
+part 'header_action.dart';
+
+/// A header widget.
+///
+/// Typically used on pages at the root of the navigation stack.
 final class FHeader extends StatelessWidget {
-  final Widget? left;
-  final Widget? right;
+  /// The title.
+  final String title;
+
+  /// The actions on the right of the header.
+  final List<Widget> actions;
+
+  /// The style.
   final FHeaderStyle? style;
 
-  const FHeader.raw({this.left, this.right, this.style, super.key});
+  /// Creates a [FHeader].
+  const FHeader({
+    required this.title,
+    this.actions = const [],
+    this.style,
+    super.key,
+  });
 
   @override
   Widget build(BuildContext context) {
     final style = this.style ?? context.theme.headerStyle;
+    final font = context.theme.font;
 
-    return Column(
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
       children: [
-        left ?? const SizedBox(),
-        right ?? const SizedBox(),
+        Expanded(
+          child: Text(
+            title,
+            overflow: TextOverflow.fade,
+            maxLines: 1,
+            softWrap: false,
+            style: style.title.withFont(font),
+          ),
+        ),
+        Row(children: actions),
       ],
     );
   }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties
+      ..add(StringProperty('title', title))
+      ..add(DiagnosticsProperty('actions', actions))
+      ..add(DiagnosticsProperty('style', style));
+  }
 }
 
+/// [FHeader]'s style.
 final class FHeaderStyle with Diagnosticable {
+  /// The title's style.
```

Comment:
```suggestion

  /// The title's style.
```

Reply:
I don't think there should be an extra space. Looking at the flutter codebase, this isn't the convention followed.